beacon-chain/blockchain/head.go

```diff
@@ -23,6 +23,21 @@ import (
 	"go.opencensus.io/trace"
 )
 
+// UpdateHeadWithBalances updates the beacon state head after getting justified balanced from cache.
+func (s *Service) UpdateHeadWithBalances(ctx context.Context) error {
+	cp := s.store.JustifiedCheckpt()
+	if cp == nil {
```

Comment:
See above ^. If we have a nil justified cpt, we return genesis, but here we return an error, which seems contradictory